### PR TITLE
Additional Task for JQuery

### PR DESCRIPTION
**Pull request by TaniaKho**
Please review my homework. Thank you!

### CODE REVIEW

File: tkhomich/08_jquery/Table.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	$(document).ready(function(){
+        $('#table_data tr').click(function(){
+            $(this).addClass('trdisplay');
+			$('.show').val('Show values!').text('Show values!');
+			});
+            $('.show').click(function(){
+            $('#table_data tr').removeClass('trdisplay');
+            $('.show').val('Rows are not hidden').text('"Rows are not hidden!');
+        });
+		$('h2').click(function(){
```
Comment (pluhin) [inline tkhomich/08_jquery/Table.html]: Why does it do here?

File: tkhomich/08_jquery/Table.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+		 <tr>
+			 <th>Крупин Сергей Иванович</th>
+			 <td>М</td> 
+			 <td>водитель 2</td>
+			 <td>01.01.1983</td>
+			 <td>3234534</td>
+			 <td>923123379</td>
+		 </tr>
+ 	</table>
+	</div>
```
Comment (pluhin) [inline tkhomich/08_jquery/Table.html]: ?

File: tkhomich/08_jquery/Python_Example.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+<HEAD>
+	<META charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+	<TITLE>Программа на Python — часть 1</TITLE>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.css">
+
+</HEAD> 
+<BODY>
+
+		<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```
Comment (pluhin) [inline tkhomich/08_jquery/Python_Example.html]: Skeleton does not have these classes

File: tkhomich/08_jquery/Passwords.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+	<meta charset="UTF-8">
+	<title>Jquery Sample</title>
+	<style>
+		.error-box{
+    color: red;
+    font-size: 12px;
+}
+	</style>
+	<script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
+	<script>
```
Comment (pluhin) [inline tkhomich/08_jquery/Passwords.html:13]: I did a few of changes, you can try
```
    <script>
    $(document).ready(function(){
        $('fieldset').keyup(function(){
            var alert_message;
            if (!$('#pass1').val() && !$('#pass2').val() ){
               alert_message="Введите значения";
            }
            else if ($('#pass1').val() == $('#pass2').val()){
                alert_message="Пароли совпадают";
            }
            else {
                alert_message="Пароли не совпадают. Попробуйте еще раз.";
            }
            $('legend').text(alert_message).addClass('error-box')
        });
    });
    </script>
```

**Review by pluhin — CHANGES_REQUESTED** [review]

File: tkhomich/08_jquery/Python_Example.html (hunk 3 above)
Comment (TaniaKho) [reply tkhomich/08_jquery/Python_Example.html]: this is example file. not related tp homework. I'll remove it.

**Comment by pluhin** [comment]
Could you please resolve conflict

**Pull request merged**